src/main/java/com/example/employee/dao/Employee.java

```diff
@@ -42,17 +41,14 @@ public static void save(EmployeeRequest employee) {
             ps.setString(7, employee.getDepartment());
             ps.executeUpdate();
             ps.close();
-        } catch (Exception e) {
-            logger.error("Error while saving data to database: ", e);
         }
     }
 
-    public static void update(EmployeeRequest employee, String id) {
+    public static int update(EmployeeRequest employee, String id) {
```

Comment:
void was fine why return int? if you need to handle the exception throw the exception itself